src/main/java/lotto/Lotto.java

```diff
@@ -1,20 +1,66 @@
 package lotto;
 
+import Enum.ErrorMessage;
+
 import java.util.List;
 
 public class Lotto {
     private final List<Integer> numbers;
+    // 매직 넘버
+    private static final int startRange = 1;
+    private static final int endRange = 45;
```

Comment:
static final 필드를 가장 먼저 정의하고 Upper snake case로 적어주는 것이 컨벤션!
```suggestion
    private static final int START_RANGE = 1;
    private static final int END_RANGE = 45;
    
    private final List<Integer> numbers;
```
[nit] 개인적으로는 start, end range보다 min, max value(or number)로 정의하는 걸 선호함 ㅋㅋ